lib/mix/torch.ex

```diff
@@ -40,6 +40,21 @@ defmodule Mix.Torch do
     %{otp_app: otp_app, format: format}
   end
 
+  def ensure_phoenix_is_loaded(mix_task) do
```

Comment:
Let's name this function as a "bang function" since it will raise an error on failure

    def ensure_phoenix_is_loaded!/1`